docs/integrations/github_app.md

```diff
@@ -0,0 +1,116 @@
+---
+title: GitHub App Integration
+category: 62cdf6722c2c1602a4b69643
+hidden: false
+---
+
+# Overview
+
+Phylum provides a GitHub App to get your current and future repositories monitored quickly and easily. Follow along here or with the [Quickstart Guide within the Phylum UI](https://app.phylum.io/quickstart/github).
+
+## Prerequisites
+
+* Ability to install Apps in GitHub
+* [Phylum account linked to GitHub](https://docs.phylum.io/docs/federate_account)
+* Phylum Account set to Pro, if additional functionality is desired
+
+## Installation Walkthrough
+
+1. Go to the [Phylum App on the GitHub Marketplace](https://github.com/marketplace/phylum-io).
+2. Click to install the free Phylum App. Be sure to select the intended account under the Account drop-down.
+
+   ![GitHub App initial install screen](https://raw.githubusercontent.com/phylum-dev/documentation/main/assets/gh_app_install_1.png)
+
+3. Fill in or edit account billing information.
+4. Click the button to "Complete order and begin installation"
+5. Choose **All** (default) or **Select** repositories and click to install. This setting controls which repositories _can be_ monitored by Phylum (i.e., visibility). Configuring repositories to be monitored [is a different step](#monitoring).
+
+   ![GitHub App install confirmation screen](https://raw.githubusercontent.com/phylum-dev/documentation/main/assets/gh_app_install_2.png)
+
+6. After installation, you will be redirected to the Phylum application to configure monitoring.
+7. If you are a new user or are not logged in, select the GitHub button at the login screen.
+   * This will automatically provision an account for new users.
+8. Congratulations - the Phylum GitHub App is installed and ready to be configured to monitor your software supply chain!
+
+## Usage
```

Comment:
Now that the on-demand analysis button has landed, we should probably have a section explaining that as well.

Reply:
Yes...no problem...change coming...